src/databricks/labs/ucx/hive_metastore/catalog_schema.py

```diff
@@ -29,10 +30,22 @@ def __init__(
     def create_all_catalogs_schemas(self, prompts: Prompts):
         candidate_catalogs, candidate_schemas = self._get_missing_catalogs_schemas()
         for candidate_catalog in candidate_catalogs:
-            self._create_catalog_validate(candidate_catalog, prompts)
+            try:
+                self._create_catalog_validate(candidate_catalog, prompts)
+            except BadRequest as e:
+                if "already exists" in str(e):
```

Comment:
is it really BadRequest?

<img width="913" alt="image" src="https://github.com/databrickslabs/ucx/assets/259697/55d6c0ec-56f9-48b7-96bf-851a13f7b3ec">

Reply:
This is the error that we are getting

`databricks.sdk.errors.platform.BadRequest: Catalog 'dev_catalog' already exists`